src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -183,4 +183,40 @@ public void cacheUpdateNewerResponse() throws Exception {
             container.stop();
         }
     }
+    /**
+     * LastModifiedCachingWire can send request directly,
+     * if it contains the "If-Modified-Since" header.
+     * @throws Exception - if the test fails
+     */
+    @Test
+    public void sendsRequestDirectly() throws Exception {
+        final MkContainer container = new MkGrizzlyContainer()
+            .next(
+                new MkAnswer.Simple(
+                    HttpURLConnection.HTTP_OK, LastModifiedCachingWireTest.BODY
+                )
+            ).next(
+                new MkAnswer.Simple(
+                    HttpURLConnection.HTTP_OK, LastModifiedCachingWireTest.BODY
+                )
+            )
+            .start();
+        try {
+            final Request req = new JdkRequest(container.home())
+                .through(LastModifiedCachingWire.class).header(
+                    HttpHeaders.IF_MODIFIED_SINCE,
+                    "Fri, 01 Jan 2016 00:00:00 GMT"
+                );
+            for (int idx = 0; idx < 2; ++idx) {
```

Comment:
@amihaiemil BTW, are you sure that Qulice will not allow just `i` here? It's more natural name for index.